### DISCUSSION TITLE
casambi-bt-hass

### DISCUSSION THREAD

**Discussion by Alexei-vk2** [lkempf/casambi-bt-hass#73] [Show and tell]
hello!
can you tell me, please, where to get the URL link to the repository?
 
https://github.com/lkempf/casambi-bt-hass
 
Installation
Manual
Place the casambi_bt folder in the custom_components folder.

HACS
Add this repository as custom repository in the HACS store (HACS -> integrations -> custom repositories):

Setup HACS https://hacs.xyz/
Go to integrations section.
Click on the 3 dots in the top right corner.
Select "Custom repositories"
Add the URL to the repository.  —  where to get the URL link to the repository?
Select the integration category.
Click the "ADD" button.
Search for the Casambi Bluetooth integration and install it.

**Comment by lkempf** [comment]
https://github.com/lkempf/casambi-bt-hass

That's the URL you need.

**Comment by Alexei-vk2** [comment]
Failed to load Setup Wizard: 500: Internal Server Error  -  how to fix it?

![Снимок экрана 2024-04-07 000230](https://github.com/lkempf/casambi-bt-hass/assets/166062625/d510b4c6-e552-42b9-a8ac-d68c86a1d7eb)

**Reply by lkempf** [reply]
That's not my integration. That's https://github.com/hellqvio86/home_assistant_casambi

**Comment by Alexei-vk2** [comment]
Hello!
Faced with this situation:
Casambi integration is installed in the home assistant.
when the "unavailable" condition occurs, the lights turn on and off.

is there any way to make sure that even after the "not available" state, the devices have the last mode so that they are turned off?
can you tell me how to do this?
![Uploading WhatsApp Image 2024-05-13 at 10.57.27.jpeg…]()

**Comment by Alexei-vk2** [comment]
Hello!
Faced with this situation:
Casambi integration is installed in the home assistant.
when the "unavailable" condition occurs, the lights turn on and off.

is there any way to make sure that even after the "not available" state, the devices have the last mode so that they are turned off?
can you tell me how to do this?
![WhatsApp Image 2024-05-13 at 10 57 27](https://github.com/lkempf/casambi-bt-hass/assets/166062625/828b425e-c3cc-42b4-a508-dcd50a773353)
